### DISCUSSION TITLE
Terminal hatası

### DISCUSSION THREAD

**Discussion by Esratass** [19/classroom#19] [Anonim] [answered]
![image](https://user-images.githubusercontent.com/92054816/158006106-629ce674-428d-4c78-8a6c-f6607a0dbe1a.png)
![image](https://user-images.githubusercontent.com/92054816/158006144-903232f7-2af9-45f2-9c64-d4135bf0d10d.png)

**Comment by gulcanyildiz** [comment, marked as answer]
[Classroom ödevleri nasıl yapılır](https://www.youtube.com/watch?v=c9cLaZeF2AY&t=6s) videosunu dikkatli izlersen kontrol etmek için:

```sh
lets grad
```
komutu kullanılır.

Amacın sadece programı çalıştırmak ise:

```sh
lets play
```
diyebilirsin.

Bu işlemi `lets` yerine manuel (elle) yapmak istersen şöyle yapacaksın (`lets play` ile hemen hemen aynı):

```sh
gcc -o assigment assigment.c
./assigment
```
